app/scripts/modules/amazon/src/loadBalancer/targetGroup.states.ts

```diff
@@ -19,21 +21,12 @@ module(TARGET_GROUP_STATES, [
     },
     views: {
       'detail@../insight': {
-        templateProvider: ['$templateCache', '$stateParams', 'cloudProviderRegistry',
-          ($templateCache: ng.ITemplateCacheService,
-           $stateParams: StateParams,
-           cloudProviderRegistry: CloudProviderRegistry) => {
-            return $templateCache.get(cloudProviderRegistry.getValue($stateParams.provider, 'loadBalancer.targetGroupDetailsTemplateUrl'));
-        }],
-        controllerProvider: ['$stateParams', 'cloudProviderRegistry',
-          ($stateParams: StateParams,
-           cloudProviderRegistry: CloudProviderRegistry) => {
-            return cloudProviderRegistry.getValue($stateParams.provider, 'loadBalancer.targetGroupDetailsController');
-        }],
-        controllerAs: 'ctrl'
+        component: TargetGroupDetails,
```

Comment:
Route to an `@Overridable` component, make sure it gets `accountId` as a prop.  It will use `accountId` to look up the proper override in the `CloudProviderRegistry`